docs-src/references/dynamic-configuration.md

```diff
@@ -0,0 +1,209 @@
+| `frontend.namespaceRPS`                | Int  | Rate limit per second applied for each Namespace.                                                                                                                                                                                  | 2400          |
+| `frontend.namespaceCount`              | Int  | Rate limit on concurrent Task Queue polls per Namespace per instance.                                                                                                                                                              | 1200          |
+| `frontend.globalNamespaceRPS`          | Int  | Namespace rate limit per second applied globally on the entire Cluster. The limit is evenly distributed among available Frontend service instances. If this is set, it overrides the per-instance limit (`frontend.namespaceRPS`). | 0             |
+| `internal-frontend.globalNamespaceRPS` | Int  | Namespace rate limit per second across all internal-frontends.                                                                                                                                                                     | 0             |
+| History                                |      |                                                                                                                                                                                                                                    |               |
+| `history.rps`                          | Int  | Request rate per second for each History host.                                                                                                                                                                                     | 3000          |
+| Matching                               |      |                                                                                                                                                                                                                                    |               |
+| `matching.rps`                         | Int  | Request rate per second for each Matching host                                                                                                                                                                                     | 1200          |
+| `matching.numTaskqueueReadPartitions`  | Int  | Number of read partitions for a Task Queue. Must be set with `matching.numTaskqueueWritePartitions`.                                                                                                                               | 4             |
+| `matching.numTaskqueueWritePartitions` | Int  | Number of write partitions for a Task Queue.                                                                                                                                                                                       | 4             |
```

Comment:
note those two configs must be updated in pair and follow a certain order to prevent impact to running workflows
When increasing # of partitions, increase reader partitions first, then write partitions
When decreasing, do it in reverse order and make sure to wait long enough before reducing # of read partitions

Reply:
Yeah; I had this [explanation here but removed it ](https://github.com/temporalio/documentation/pull/2029/files/18f50fdced2133939f68ec82f12b6dc343b2ee54..1c715efb4feb5c4da6293a5a48c4df76c373b4a1#r1221959428) because this doc is a reference. I can explain this better in the next round of updates as I build out the OSS Cluster deployment guide.